docs/04-api-reference/front-commerce-core/create-resized-image-response.mdx

```diff
@@ -6,8 +6,9 @@ description:
   resizing and caching images on the fly."
 ---
 
-The `createResizedImageResponse` allows to create [resource routes](https://remix.run/docs/en/1.19.3/guides/resource-routes) which allow
-resizing and caching images on the fly.
+The `createResizedImageResponse` allows to create
+[resource routes](https://remix.run/docs/en/1.19.3/guides/resource-routes) which
```

Comment:
**question:** is it wanted to link to Remix 1.19.3 doc?

**suggestion:** link to the latest version

```suggestion
[resource routes](https://remix.run/docs/en/2.0.1/guides/resource-routes) which
```